src/annotator/guest.js

```diff
@@ -361,7 +369,16 @@ export default class Guest {
     });
   }
 
+  /**
+   * Attempt to connect to the sidebar frame.
+   */
+  async _connectToSidebar() {
```

Comment:
There is a reference to the public `connectToSidebar` method in the description of the `Guest` class that needs to be removed. You could add a note in the docs for the constructor that the Guest will connect to sidebar as part of its initialization.